### DISCUSSION TITLE
v1.4.3 - Couple of platform specific bug fixes and Docker image

### DISCUSSION THREAD

**Discussion by manugarg** [manugarg/pacparser#186] [Announcements]
## What's Changed
* Add basic support for FreeBSD by @ehaupt in https://github.com/manugarg/pacparser/pull/171
* Fix memory leak in pacparser_setmyip by @manugarg in https://github.com/manugarg/pacparser/pull/174
* Open pac file in rb mode. by @manugarg in https://github.com/manugarg/pacparser/pull/175
* Hash pin workflows and enable dependabot by @joycebrum in https://github.com/manugarg/pacparser/pull/177
* [makefile] Use va_copy everywhere instead of __va_copy by @manugarg in https://github.com/manugarg/pacparser/pull/181
* [docker] Add docker image build by @manugarg in https://github.com/manugarg/pacparser/pull/182

## New Contributors
* @ehaupt made their first contribution in https://github.com/manugarg/pacparser/pull/171
* @dependabot made their first contribution in https://github.com/manugarg/pacparser/pull/178

**Full Changelog**: https://github.com/manugarg/pacparser/compare/v1.4.2...v1.4.3

<hr /><em>This discussion was created from the release <a href='https://github.com/manugarg/pacparser/releases/tag/v1.4.3'>v1.4.3 - Couple of platform specific bug fixes and Docker image</a>.</em>

**Comment by tech189** [comment]
Hi, just to let you know, the binaries are missing from this release, e.g. https://github.com/manugarg/pacparser/releases/download/v1.4.2/pacparser-v1.4.2-windows.zip for v1.4.2.

**Reply by manugarg** [reply]
Thanks for the reminder, @tech189! I've added the binaries now.
